include/exadg/structure/spatial_discretization/operator.h

```diff
@@ -331,6 +342,10 @@ class Operator : public dealii::Subscriptor, public Interface::Operator<Number>
   unsigned int
   get_quad_index_gauss_lobatto() const;
 
+  double
```

Comment:
Here, a comment would be useful saying that this function assumes a damping operator that is realized as a scaled mass matrix.